src/cdk/drag-drop/drag.ts

```diff
@@ -491,6 +491,12 @@ export class CdkDrag<T = any> implements AfterViewInit, OnDestroy {
       // Preserve the previous `transform` value, if there was one.
       this._rootElement.style.transform = this._initialTransform ?
           this._initialTransform + ' ' + transform : transform;
+
+      // Apply transform as attribute if dragging and svg element to work for IE
+      if (this._rootElement instanceof SVGElement) {
```

Comment:
While it's unlikely, this has the potential of throwing an error during server-side rendering. Changing it to `typeof SVGElement !== 'undefined' && this._rootElement instanceof SVGElement` should avoid the issue.

Reply:
Done